### ISSUE TITLE
Installation Done - I can not even start mining - it hangs after clicking i got this.

### ISSUE THREAD

**Issue by imrecsoka** [BobbyGR/AIOMiner#104]
Hi,

Running windows 10 wth 1 nvidia and 1 amd card- i have installed AIO - done.
I have tried beginner and i got this mode, same thing, just hangs.
Any recommendation ?

**Comment by BobbyGR** [comment]
Ethash is the only algo that we support with two different brands.  What are you trying to mine?  What do you see?  Enabled debug mode and hit start?

**Comment by imrecsoka** [comment]
![image](https://user-images.githubusercontent.com/25494601/65391791-c8ce1b80-dd9f-11e9-8646-738371117c8b.png)

This is what i see, i just see the "splash screen" i have tried beginner mode where i chose bitcoingold, i put in my address and i was expecting to see it start mine.
I then tried i got this mode, and it does the same thing, i click it and it doesnt no anything.

**Comment by BobbyGR** [comment]
Hmm I'll try to duplicate.  Try clicking on 'i got this' instead.  Then you can go-to pool settings, find Bitcoin gold, select one of our pools preloaded, put in your address and hit save.  Exit, then from the main drop-down on 'coin to mine' select butcoing gold, hit start

**Comment by imrecsoka** [comment]
Hi Bobby, please sort me out man, log  to my machine or somthing.
I have managed to get in to mining facility  but it very very slowto get in, i dont understand why the hell it is that way. I dont have app issues with nicehash or multipoolhub miner 
Anyways, lets start from scratch...how do we do this?
I have placed my AIOMiner.exe file in C:\AIOMiner folder
When i run it , and even with administrator... it doesnt launch, what is wrong with this?
Can we please try start afresh - u are welcome to log in to my machine also.

**Comment by BobbyGR** [comment]
Click on Support on AIOMiner.com to join the discord for faster support.  You should be installing to 'my documents' to avoid the admin issue.  I can't actually log into your machine without some written consent contract as once I do you could blame us for everything under the sun.  So generally, we don't do it.  Please join the discord chat and we can walk through it.

**Comment by imrecsoka** [comment]
uninstalled alll antivirus manually disabled windows defender bullshit also.
deleted all the aiominer stuff
ran setup with admin rights and ran the prof from the desktop icon as admin again
i chose beginner mode - it seems for me ethereum classic is not possible ?
i have a 3gb nvidia card gtx 1060 and a rx 470 4gb .... u can tell me.... but the claymore miner says there is a dag file size issue. I have tried to run the dagfix program i found on a youtub channel, but i dont think in windows 10 and because of DAG file size , it is possible to mine with a 3GB nvidia card?
Then again not sure for a 4GB RX470- i would imagine this is fine...?
my log folder is blank  so i dont know hat i am supposed to send you.

I repeated the process above - chose beginner and then selected bitoin gold:
It keeps saying there is a miner missing and it downloads it to start?
no idea... it is clearly not doing this.

![image](https://user-images.githubusercontent.com/25494601/65539751-a291b280-df3c-11e9-965e-7f0a895a2d8e.png)
